diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 15a370b2d3d52929966ea2398a07069b51d42d1f..a8ce10c2a973c126bd1590fea296a71edc58cfc2 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -329,6 +329,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
