diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 27c0162dd1016af023a2867a526a61afe948a01d..17a707f57ea311ec65963f274b64c6c33ecf8132 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2692,6 +2692,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
