diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 7338b3e02d9fe405487597136c41a6f98be8cce8..c393438c72949247122e6579abcc9e4892582ceb 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2691,6 +2691,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
