diff --git a/chrome/browser/download/download_target_determiner.cc b/chrome/browser/download/download_target_determiner.cc
index 67dfef6bd7d3b45fcf7c09288ed0438aea1f757d..7d427a83b2bef5961b43cf06d001dad2aa791f5f 100644
--- a/chrome/browser/download/download_target_determiner.cc
+++ b/chrome/browser/download/download_target_determiner.cc
@@ -1270,6 +1270,7 @@ DownloadFileType::DangerLevel DownloadTargetDeterminer::GetDangerLevel(
       download_->GetDownloadSource() != download::DownloadSource::DRAG_AND_DROP;
   if (HasPromptedForPath() ||
       confirmation_reason_ != DownloadConfirmationReason::NONE ||
+      BRAVE_DOWNLOAD_TARGET_DETERMINER_GET_DANGER_LEVEL
       user_approved_path) {
     return DownloadFileType::NOT_DANGEROUS;
   }
@@ -1309,6 +1310,7 @@ DownloadFileType::DangerLevel DownloadTargetDeterminer::GetDangerLevel(
         ui::PAGE_TRANSITION_FROM_ADDRESS_BAR) != 0 ||
        (download_->HasUserGesture() && visits == VISITED_REFERRER)))
     return DownloadFileType::NOT_DANGEROUS;
+  BRAVE_DOWNLOAD_TARGET_DETERMINER_GET_DANGER_LEVEL2
   return danger_level;
 }
 
