diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 0dced3bcbdc1d1b8fed6ebed6b99d0dd0002787c..d9bbf964312a15fb2785cc467eaabb5269c532eb 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2691,6 +2691,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
