diff --git a/chrome/browser/ui/browser_command_controller.cc b/chrome/browser/ui/browser_command_controller.cc
index 5664d44624283597c6a6b564badf541e2bac2fd4..f65fd30c7059e707043d1377623f48075093e5ce 100644
--- a/chrome/browser/ui/browser_command_controller.cc
+++ b/chrome/browser/ui/browser_command_controller.cc
@@ -228,7 +228,7 @@ BrowserCommandController::BrowserCommandController(Browser* browser)
       prefs::kDownloadRestrictions,
       base::BindRepeating(&BrowserCommandController::UpdateSaveAsState,
                           base::Unretained(this)));
-#if !BUILDFLAG(IS_MAC)
+#if !BUILDFLAG(IS_MAC) && !BUILDFLAG(IS_WIN) && !BUILDFLAG(IS_LINUX)
   profile_pref_registrar_.Add(
       prefs::kFullscreenAllowed,
       base::BindRepeating(
