diff --git a/components/content_settings/core/browser/content_settings_uma_util.cc b/components/content_settings/core/browser/content_settings_uma_util.cc
index e445bb768a6d1a0d2565647b6d9c98c169d735ab..87ea5be329d408b04eac30bd3a4a0e43327aab09 100644
--- a/components/content_settings/core/browser/content_settings_uma_util.cc
+++ b/components/content_settings/core/browser/content_settings_uma_util.cc
@@ -134,6 +134,7 @@ constexpr auto kHistogramValue = base::MakeFixedFlatMap<ContentSettingsType,
 
     // As mentioned at the top, please don't forget to update ContentType in
     // enums.xml when you add entries here!
+    BRAVE_HISTOGRAM_VALUE_LIST
 });
 
 constexpr int kkHistogramValueMax =
