diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 55765cb73b3e4c41ed785a9da8906741e62f4a21..125314d253ec610c6be6a54007e8aeba69db2370 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -69,6 +69,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
