diff --git a/crypto/unexportable_key_metrics.cc b/crypto/unexportable_key_metrics.cc
index e0f9ee70c1bdd2cfec3df2df837f015ef1f83a98..8d07b54ca24262d5631cfdc7f21e2b1ca3aada09 100644
--- a/crypto/unexportable_key_metrics.cc
+++ b/crypto/unexportable_key_metrics.cc
@@ -28,6 +28,7 @@ enum class KeyType {
 };
 
 const SignatureVerifier::SignatureAlgorithm kAllAlgorithms[] = {
+    BRAVE_ALL_ALGORITHMS
     SignatureVerifier::SignatureAlgorithm::ECDSA_SHA256,
     SignatureVerifier::SignatureAlgorithm::RSA_PKCS1_SHA256,
 };
@@ -137,6 +138,7 @@ internal::TPMSupport MeasureVirtualTpmOperations() {
   if (algo) {
     switch (*algo) {
       case SignatureVerifier::SignatureAlgorithm::ECDSA_SHA256:
+      BRAVE_MEASURE_VIRTUAL_TMP_OPERATIONS
         supported_virtual_algo = internal::TPMSupport::kECDSA;
         break;
       case SignatureVerifier::SignatureAlgorithm::RSA_PKCS1_SHA256:
@@ -216,6 +218,7 @@ void MeasureTpmOperationsInternal(UnexportableKeyProvider::Config config) {
   if (algo) {
     switch (*algo) {
       case SignatureVerifier::SignatureAlgorithm::ECDSA_SHA256:
+      BRAVE_MEASURE_TMP_OPERATIONS_INTERNAL
         supported_algo = internal::TPMSupport::kECDSA;
         break;
       case SignatureVerifier::SignatureAlgorithm::RSA_PKCS1_SHA256:
