diff --git a/extensions/browser/api/web_request/extension_web_request_event_router.cc b/extensions/browser/api/web_request/extension_web_request_event_router.cc
index 7b99af2d533f222235eeddcf8b2b24065d85de0e..ba4c78165c5a3428962ec18b428c1920c30a5188 100644
--- a/extensions/browser/api/web_request/extension_web_request_event_router.cc
+++ b/extensions/browser/api/web_request/extension_web_request_event_router.cc
@@ -1209,6 +1209,7 @@ WebRequestEventRouter::OnAuthRequired(content::BrowserContext* browser_context,
                                          *request)) {
     return AuthRequiredResponse::AUTH_REQUIRED_RESPONSE_NO_ACTION;
   }
+  BRAVE_EXTENSION_WEB_REQUEST_EVENT_ROUTER_ON_AUTH_REQUIRED
 
   int extra_info_spec = 0;
   RawListeners listeners = GetMatchingListeners(
